apple/kernel-modules/bridge-drv.nix

```diff
@@ -0,0 +1,40 @@
+{ pkgs, kernel }:
+
+pkgs.stdenv.mkDerivation rec {
+  name = "mbp-bridge";
```

Comment:
I think this is a good candidate for nixpkgs.

Reply:
ok, will more it